src/lib/upload-forms.js

```diff
@@ -39,6 +39,7 @@ module.exports = async (projectDir, subDirectory, options) => {
     }
 
     const xml = fs.read(xformPath);
+    const xmlVersion = fs.statSync(filePath).mtime;
```

Comment:
I'm not sure Git will preserve this correctly.

Reply:
Yeah it looks like it doesn't preserve it. Given this I think we should instead... when building the form in cht-conf, if the output is different than the previous version, store the current time somewhere in a source controlled file.

There are some challenges with this approach... for example

1. If you repo is out of date then it may create a spurious version but spurious versions aren't the end of the world as long as everyone uses less-than and greater-than comparisons.
2. If the system clock is out of date the version will jump around. This should be rare especially in a CICD setup.

Reply:
If you are okay adding a Git library, then we could use the Git mod time,

```
$ git log -1 --pretty="format:%ci" test/data/lib/upload-forms/merge-properties/forms/example.xml
2020-04-14 21:54:42 +1200
```